pages/common/scamper.md

```diff
@@ -0,0 +1,29 @@
+# scamper
+
+> A tool that actively probes the Internet in order to analyze topology and performance.
```

Comment:
```suggestion
> Actively probe the Internet in order to analyze topology and performance.
```

Reply:
> Like its predecessor [skitter](https://www.caida.org/catalog/software/skitter/), scamper is a feature-rich tool that actively probes the Internet in order to analyze topology and performance.

Here I have "stolen" the text from the [website](https://www.caida.org/catalog/software/scamper/).

Reply:
Yeah, I realized it, and that's OK, but it's recommended that our descriptions follow the style guide. "A tool that" is kinda useless, many commands could add that to the beggining of the description and that wouldn't add any value. We have some specific rules in our [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood) that makes all descriptions standardized:
> ### Imperative Mood
> 
> - **All descriptions must be concise and phrased in the imperative mood.**
> - This also applies to all translations by default unless otherwise specified in the language-specific section below.
> - For example, when writing documentation for `cd`, a tool to check out and work on a specific directory in the Terminal or Command Prompt, **do not** write a lengthy description such as:
> 
> ```md
> > `cd` is a system tool, available in Windows, macOS, and Linux, to check out a specific directory to get things done in the Command Prompt, Terminal, and PowerShell.
> ```
> 
> It should instead be simplified to make it easier for everyone to read:
> 
> ```md
> > Change the current working directory.
> ```